### PR TITLE
Fixed textTooLink warnings

### PR DESCRIPTION
**Pull request by K42IE**
Removed textToLink component (will make issue to track). 

+ Added functionality to scroll for each button

Note: Design team has not created a solution to this. I only fixed the engineering side of the issue

### CODE REVIEW

**Comment by K42IE** [comment]
Issue to track deletion: https://github.com/gosqasorg/asset-provenance-tracking/issues/176

**Comment by devhawk** [comment]
looks good, but holding off approval until #169 is merged

**Review by devhawk — APPROVED** [review]
tested locally, looks good. Merging

**Pull request merged**
